### DISCUSSION TITLE
Comentarios de Documentación

### DISCUSSION THREAD

**Discussion by Nikzza** [INGCOM-UNRN-P1/cursada-2023#109] [Preguntas y respuestas] [answered]
queria consultar si los contadores o alguna variable inicializada DENTRO de la funcion tienen que ser declaradas como parametros o como se declararian?

**Comment by martinvilu** [comment, marked as answer]
Solo los parámetros y retornos, en la clase del último miércoles tratamos este tema.

**Reply by martinvilu** [reply]
> en la clase del último miércoles tratamos este tema.

Para mas detalles
